composer.json

```
@@ -9,7 +9,7 @@
     ],
     "require": {
         "php": "^7.4 || ^8.0",
-        "sylius/sylius": "~1.9.0 || ~1.10.0",
+        "sylius/sylius": "~1.9.0 || ~1.10.0 || ~1.11.0",
```

```suggestion
        "sylius/sylius": "^1.9",
```
It will avoid adding futur versions here until 2.0 will be release

---

I'm in doubt about this one. The downside is that CI does not upgrade automatically in a similar way, so we still have to add it to the build matrix each time. Although on the other hand it might also be handy to spot issues earlier, as users will upgrade their installations.

---

Majority of plugins are acting like that, you will block some upgrade plans if you try to list all new versions.
Right now for example, I'm forced to make my own fork of this plugin to continue to upgrade to Sylius 1.11.

If a dev is starting to develop with Sylius (latest version), he won't be allowed to tests your plugin because of this requirement.

Letting devs test the untested Sylius version is not a big deal I guess. If there is bug with this new version, then devs will likely create an issue and maybe helps you by making a PR.

---

Yep - makes sense. I'll adjust it after this PR :-)